internal/commands/user_test.go

```diff
@@ -107,3 +109,145 @@ func buildSlackMock() *slack.User {
 		},
 	}
 }
+
+func TestGetStringInt64(test *testing.T) {
+	scenarios := []struct {
+		name         string
+		input        string
```

Comment:
What do you think of calling this something like `stringValue`, instead `input`?

Reply:
I see no reason to change that. I am following the project pattern, for example:

```golang
type userCommandFixture struct {
	testName                string
	expectError             bool
	errorMessage            string
...
```

Test scenarios for me seem to make more sense to have input and output, it will not always be the same type.

Maybe I'm wrong, is there a pattern for that?

Reply:
There is no pattern yet, in my tests I was declaring the variables with a descriptive name, but I like this idea of an input and an output, that type changes for each test.
Make sense.